Open Judge System/OJS.Common/Extensions/DirectoryHelpers.cs

```diff
@@ -30,5 +46,32 @@ public static void SafeDeleteDirectory(string path, bool recursive = false)
                 Directory.Delete(path, recursive);
             }
         }
+
+        public static void DeleteExecutionStrategiesWorkingDirectories()
+        {
+            var directoryPaths = Directory.GetDirectories(GlobalConstants.ExecutionStrategiesWorkingDirectoryPath);
+            foreach (var dirPath in directoryPaths)
+            {
+                var dir = new DirectoryInfo(dirPath);
+                if (dir.Exists && dir.CreationTime < DateTime.Now.AddHours(-1))
+                {
+                    var isDeleted = false;
+                    var retryCount = 0;
+                    while (!isDeleted && retryCount <= 3)
+                    {
+                        try
+                        {
+                            SafeDeleteDirectory(dirPath, true);
```

Comment:
`Save...`

Reply:
`Safe` as in "safely" (adverb). Save cannot be used as imperative/verb doesn't make sense being used before another verb.